src/sentry/apidocs/parameters.py

```diff
@@ -17,6 +18,50 @@ class GLOBAL_PARAMS:
         type=str,
         location="path",
     )
+    STATS_PERIOD = OpenApiParameter(
+        name="statsPeriod",
+        location="query",
+        required=False,
+        type=str,
+        description="""The period of time for the query, will override the start & end parameters, a number followed by one of:
+- `d` for days
+- `h` for hours
+- `m` for minutes
+- `s` for seconds
+- `w` for weeks
+
+For example `24h`, to mean query data starting from 24 hours ago to now.""",
+    )
+    START = OpenApiParameter(
+        name="start",
+        location="query",
+        required=False,
+        type=OpenApiTypes.DATETIME,
+        description="The start of the period of time for the query, expected in ISO-8601 format. For example 2001-12-34T12:34:56.7890",
```

Comment:
```suggestion
        description="The start of the period of time for the query, expected in ISO-8601 format. For example `2001-12-14T12:34:56.7890`",
```
I think `2001-12-34T12:34:56.7890` might be an invalid date due to 34th day 😆